vcr/vcr.go

```diff
@@ -225,7 +226,16 @@ func (c *vcr) Configure(config core.ServerConfig) error {
 		c.openidSessionStore = c.storageClient.GetSessionDatabase()
 	}
 
-	status := revocation.NewStatusList2021(c.storageClient.GetSQLDatabase(), client.NewWithCache(config.HTTPClient.Timeout))
+	status := revocation.NewStatusList2021(c.storageClient.GetSQLDatabase(), client.NewWithCache(config.HTTPClient.Timeout),
+		func(issuer did.DID, page int) (string, error) {
```

Comment:
Any reason to make the URL resolution dynamic in this way? Are we going to magically configure any other implementation here?

Reply:
Yes, but probably not needed. Simplified it with simply the base URL.